src/AppBundle/Entity/Traits/ActiveAudit.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace AppBundle\Entity\Traits;
+
+/**
+ * ActiveAudit Trait, usable with PHP >= 5.4
+ *
+ */
+trait ActiveAudit
+{
+    /**
+     * Get date that this entity was active from
+     *
+     * @var \DateTime
+     * @JMS\Type("DateTime<'Y-m-d H:i:s'>")
+     * @JMS\Groups({"active-period"})
+     */
+    private $activeFrom;
+
+    /**
+     * Get date that this entity was active to
+     *
+     * @var \DateTime
+     *
+     * @JMS\Type("DateTime<'Y-m-d H:i:s'>")
+     * @JMS\Groups({"active-period"})
+     */
+    private $activeTo;
+
+    /**
+     * @return \DateTime
+     */
+    public function getActiveFrom()
+    {
+        return $this->activeFrom;
+    }
+
+//    /**
+//     * @param \DateTime $activeFrom
+//     *
+//     * @return $this
+//     */
+//    public function setActiveFrom(\DateTime $activeFrom)
+//    {
+//        $this->activeFrom = $activeFrom;
+//        return $this;
+//    }
```

Comment:
Should remove this code rather than commenting it out

Reply:
Done